src/notebooks/finance/interventions/kamiran.ipynb

```diff
@@ -367,7 +371,7 @@
     "        for sex in range(2)\n",
     "    ]\n",
     ")\n",
-    "dp_bar"
+    "dp_bar.show()"
```

Comment:
Better to remove `.show()`, it opens browser tabs when running the build locally.